File: newDemo/newDemo/Base/ChooseOperate/cell/ChooseOperateTableViewCell.m
//
//  ChooseOperateTableViewCell.m
//  BaseToolSystem
//
//  Created by 靳建南 on 2018/11/6.
//  Copyright © 2018年 靳建南. All rights reserved.
//

#import "ChooseOperateTableViewCell.h"

@implementation ChooseOperateTableViewCell
{
    UILabel * _contentLab;
}
-(id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier{
    if (self = [super initWithStyle:style reuseIdentifier:reuseIdentifier]) {
        
        //title
        _titleLab = [[UILabel alloc]init];
        //img
        _chooseImg = [[UIImageView alloc]init];
    }
    return self;
}
-(void)setContent{
    _titleLab.frame = CGRectMake(10, 0, ScreenW-80, 45);
    _titleLab.font = [UIFont systemFontOfSize:14];
    _titleLab.textColor = LAB1_COLOR;
    _titleLab.textAlignment = NSTextAlignmentLeft;
    [self.contentView addSubview:_titleLab];
    
    _chooseImg.frame = CGRectMake(ScreenW-70-27/2, 45/2-20/4, 27/2, 20/2);
    [self.contentView addSubview:_chooseImg];
    
    //分割线
    UILabel * bottomLineLab = [[UILabel alloc]init];
    bottomLineLab.frame = CGRectMake(10, 45-0.5, ScreenW-80, 0.5);
    bottomLineLab.backgroundColor = RGBACOLOR(225, 225, 225, 1);
    [self.contentView addSubview:bottomLineLab];
}

@end

File: BaseToolClass/BaseToolClass/Base(基类)/ChooseOperate/cell/ChooseOperateTableViewCell.swift
//
//  ChooseOperateTableViewCell.swift
//  BaseToolClass
//
//  Created by 靳建南 on 2018/11/20.
//  Copyright © 2018年 靳建南. All rights reserved.
//

import UIKit

class ChooseOperateTableViewCell: BaseTableViewCell {

    var _titleLab:UILabel!
    var _chooseImg:UIImageView!
    
    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
        super.init(style: style, reuseIdentifier: reuseIdentifier)
        _titleLab = UILabel.labWithFrame(frame: CGRect.init(x: 10, y: 0, width: ScreenW-80, height: 45), text: "", font: 13, textColor: LAB1_COLOR, textAlignment: .left)
        self.contentView.addSubview(_titleLab)
        
        _chooseImg = UIImageView.imageViewWithFrame(frame: CGRect.init(x: ScreenW-70-27/2, y: 45/2-20/4, width: 27/2, height: 20/2), imageName: "", contentMode: 0)
        self.contentView.addSubview(_chooseImg)
        
        let lineView = UIView.viewWithFrame(frame: CGRect.init(x: 10, y: 45-0.5, width: ScreenW-80, height: 0.5), backgroundColor: LINE_COLOR)
        self.contentView.addSubview(lineView)
    }

    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        print(#function)
    }
}
